<!doctype html>
<html>

<head>

  <title>
    
      *Paraphelidium* transcriptome: some code | Xavier Grau-Bové
    
  </title>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta charset="utf-8">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="stylesheet" href="/assets/css/syntax.css">
  <!-- Use Atom -->
  <link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Xavier Grau-Bové" />
  <!-- RSS-v2.0
  <link href="/rss-feed.xml" type="application/rss+xml" rel="alternate" title="Xavier Grau-Bové | Bioinformatician & evolutionary genomicist"/>  //-->

  <!-- This loads something else, I think  -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">

  <!-- This loads icons (orcid, twitter, etc.) -->
  <script type="text/javascript" async
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
  </script>

  <!-- Google Analytics -->
  <!-- <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>
 -->

  <!-- Use Jekyll SEO plugin -->
  <!-- Begin Jekyll SEO tag v2.6.1 -->
<title>Paraphelidium transcriptome: some code | Xavier Grau-Bové</title>
<meta name="generator" content="Jekyll v3.9.0" />
<meta property="og:title" content="Paraphelidium transcriptome: some code" />
<meta name="author" content="xgrau" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Paraphelidium transcriptome: some code" />
<meta property="og:description" content="Paraphelidium transcriptome: some code" />
<link rel="canonical" href="http://localhost:4000/science/paraphelidium-code.html" />
<meta property="og:url" content="http://localhost:4000/science/paraphelidium-code.html" />
<meta property="og:site_name" content="Xavier Grau-Bové" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2019-02-11T00:00:00+01:00" />
<script type="application/ld+json">
{"@type":"BlogPosting","url":"http://localhost:4000/science/paraphelidium-code.html","headline":"Paraphelidium transcriptome: some code","dateModified":"2019-02-11T00:00:00+01:00","datePublished":"2019-02-11T00:00:00+01:00","author":{"@type":"Person","name":"xgrau"},"mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/science/paraphelidium-code.html"},"description":"Paraphelidium transcriptome: some code","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


</head>


<body>

<div class="container">
  <header class="site-header">
  <h1 class="site-title">
    <a href="/"> 
		<img src="/assets/img/logo_bo2.png" alt="logo" width="30" height="30" align="left" class="verticalbottom"> Xavier Grau-Bové
	</a>
  </h1> 
  <nav class="menu-list">
    
      <a href="/pages/cv.html" class="menu-link">CV</a>
    
      <a href="/pages/data.html" class="menu-link">Data & code</a>
    
      <a href="/pages/blog.html" class="menu-link">Blog</a>
    

    
      <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
    
      <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
    
      <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
    
  </nav>
  <div class="dropdown">
    <button class="dropbtn"><i class="fa fa-bars" aria-hidden="true"></i></button>
    <div class="dropdown-content">
      
        <a href="/pages/cv.html" class="menu-link">CV</a>
      
        <a href="/pages/data.html" class="menu-link">Data & code</a>
      
        <a href="/pages/blog.html" class="menu-link">Blog</a>
      

      
        <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
      
        <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
      
        <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
      
    </div>
  </div>
</header>

  <div class="page-content">
  <h2>
    *Paraphelidium* transcriptome: some code
  </h2>
  
  <p>
  <span class="post-date">
    
    
    February
    11th,
    2019
    by
    
      xgrau
    
  </span>
  </p>

  

  <article>
    <p>This is a quick post to highlight the publication of <a href="https://www.nature.com/articles/s42003-018-0235-z">Global transcriptome analysis of the aphelid <em>Paraphelidium tribonemae</em> supports the phagotrophic origin of fungi</a> by Guifré Torruella, Purificación López-García (Université Paris-Sud) et al., in Communications Biology.</p>

<p>In parallel, we released the <code class="language-plaintext highlighter-rouge">R</code> code we used to analyse the functional profile of its gene content, which you can find it in this <strong><a href="https://github.com/xgrau/paraphelidium2018">Github repository</a></strong>. We used the absence/presence profile of a set of genes linked to primary metabolism in the genomes (or transcriptomes) of unicellular eukaryotes to highlight similarities between the gene content of fungi and <em>Paraphelidium</em>.</p>

<p>Thus, it can assist in the production of plots such as the following (<a href="https://www.nature.com/articles/s42003-018-0235-z/figures/3">Figure 3 in the paper</a>):</p>

<figure class="image">
  <div align="center"> 
    <img src="../assets/img/paraphelidium_fig3.png" alt="a) PCoA (Principal Coordinate Analysis) of gene presence for orthologs related to primary metabolism, across 41 eukaryotes. b) same data in a binary presence/absence heatmap, with species clustering" />
    <font color="gray">
    <small>
      <figcaption>a) PCoA (Principal Coordinate Analysis) of gene presence for orthologs related to primary metabolism, across 41 eukaryotes. b) same data in a binary presence/absence heatmap, with species clustering</figcaption>
    </small>
    </font> 
  </div>
</figure>

<p>The script <code class="language-plaintext highlighter-rouge">dimreduction-heatmaps_Torruella_et_al_2018.R</code> takes as input a tab-separated table where rows are genes/orthogroups and columns are various species. You can load any table you wish by editing these lines, right at the beginning of the script:</p>

<div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="n">input</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="s2">"primary_transposed.txt"</span><span class="w"> </span><span class="c1"># assumes a wide-style matrix with headers, rows=genes/OGs &amp; cols=sps </span><span class="w">
</span></code></pre></div></div>
<div class="language-R highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="n">mi</span><span class="w"> </span><span class="o">=</span><span class="w"> </span><span class="n">read.table</span><span class="p">(</span><span class="n">input</span><span class="p">,</span><span class="n">header</span><span class="o">=</span><span class="nb">T</span><span class="p">,</span><span class="n">sep</span><span class="o">=</span><span class="s2">"\t"</span><span class="p">)</span><span class="w">
</span></code></pre></div></div>

<p>Our results highlight the similarities between the rich primary metabolism of Paraphelidium and ‘canonical’ fungi. Thus, unlike other early-branching fungal allies such as <em>Rozella</em>, <em>Paraphelidium</em> does not have a simplified metabolism. This is consistent with what is actually the most interesting result of the paper (in my opinion): the phylogenomic analysis of <em>Paraphelidium</em> consolidates its position as sister-group to fungi and breaks up its association with <em>Rozella</em> and microsporidians. This has important implications regarding the lifestyle of the ancestral fungi, which was likely phagotrophic (instead of osmotrophic).</p>

<p>But you should definetely read the <a href="https://www.nature.com/articles/s42003-018-0235-z">paper</a> to get the whole picture ;)</p>

  </article>

  <p>
  <div class="post-share">
    <div class="post-date">Feel free to share!</div>
    <div class="sharing-icons">
      <a href="https://twitter.com/intent/tweet?text=*Paraphelidium* transcriptome: some code&amp;url=https://github.com/pages/xgrau/xgrau.github.io/science/paraphelidium-code.html" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
    </div>
  </div>
  </p>
    
  

</div>

  <footer class="footer">
  
    <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
  
    <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
  
    <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
  
  <div class="post-date"><a href="/">Xavier Grau-Bové</a> | <a href="https://lenpaul.github.io/Millennial/">Millennial theme by Paul Le</a></div>
</footer>

</div>

</body>
</html>
