<!doctype html>
<html>

<head>

  <title>
    
      Data & code | Xavier Grau-Bové
    
  </title>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta charset="utf-8">

  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="stylesheet" href="/assets/css/syntax.css">
  <!-- Use Atom -->
  <link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Xavier Grau-Bové" />
  <!-- RSS-v2.0
  <link href="/rss-feed.xml" type="application/rss+xml" rel="alternate" title="Xavier Grau-Bové | Bioinformatician & evolutionary genomicist"/>  //-->

  <!-- This loads icons (orcid, twitter, etc.) -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/fontawesome.min.css">
  
  <!-- THIS URL WORKS BUT HAS NO ORCID --> 
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

  <!-- This loads something else, I think  -->
  <!-- <script type="text/javascript" async
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
  </script> -->

  <!-- Google Analytics -->
  <!-- <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>
 -->

  <!-- Use Jekyll SEO plugin -->
  <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Data &amp; code | Xavier Grau-Bové</title>
<meta name="generator" content="Jekyll v3.9.0" />
<meta property="og:title" content="Data &amp; code" />
<meta name="author" content="xgrau" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Bioinformatician &amp; evolutionary genomicist" />
<meta property="og:description" content="Bioinformatician &amp; evolutionary genomicist" />
<link rel="canonical" href="http://localhost:4000/pages/data.html" />
<meta property="og:url" content="http://localhost:4000/pages/data.html" />
<meta property="og:site_name" content="Xavier Grau-Bové" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Data &amp; code" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"xgrau"},"description":"Bioinformatician &amp; evolutionary genomicist","headline":"Data &amp; code","url":"http://localhost:4000/pages/data.html"}</script>
<!-- End Jekyll SEO tag -->


</head>


<body>

<div class="container">
  <header class="site-header">
  <h1 class="site-title">
    <a href="/"> 
		<img src="/assets/img/logo_bo3.png" alt="logo" width="32" height="32" align="left" class="headerlogo"> Xavier Grau-Bové
	</a>
  </h1> 
  <nav class="menu-list">
    
      <a href="/pages/cv.html" class="menu-link">CV</a>
    
      <a href="/pages/data.html" class="menu-link">Data & code</a>
    
      <a href="/pages/blog.html" class="menu-link">Blog</a>
    

    
      <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
    
      <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
    
      <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
    
      <a href="feed.xml" class="menu-link" target="_blank"><i class="fa fa-rss-square" aria-hidden="true"></i></a>
    
  </nav>
  <div class="dropdown">
    <button class="dropbtn"><i class="fa fa-bars" aria-hidden="true"></i></button>
    <div class="dropdown-content">
      
        <a href="/pages/cv.html" class="menu-link">CV</a>
      
        <a href="/pages/data.html" class="menu-link">Data & code</a>
      
        <a href="/pages/blog.html" class="menu-link">Blog</a>
      

      
        <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
      
        <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
      
        <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
      
        <a href="feed.xml" class="menu-link" target="_blank"><i class="fa fa-rss-square" aria-hidden="true"></i></a>
      
    </div>
  </div>
</header>

  <div class="page-title misc-content">
  <h1>
    Data & code
  </h1>

<article>
  <h2 id="how-to-cite">How to cite</h2>

<p>You can cite the relevant papers (linked below, for each piece of data). Please &amp; thank you.</p>

<h2 id="orthology-inference-with-possvm">Orthology inference with <em>Possvm</em></h2>

<p><strong><em>Possvm</em></strong> is a python utility that identifies pairs and clusters of orthologous genes from gene phylogenies <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab234/6342420" target="_blank">(Grau-Bové and Sebé-Pedrós, MBE 2021)</a>. It takes advantage of the <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2007-8-6-r109">species overlap algorithm</a> implemented in the <a href="http://etetoolkit.org/docs/latest/tutorial/tutorial_phylogeny.html#species-overlap-so-algorithm"><em>ETE</em> toolkit</a> to parse the phylogeny and identify orthologous gene pairs, and <a href="https://micans.org/mcl/"><em>MCL</em> clustering</a> for orthogroup identification.</p>

<p><img align="centre" width="240" src="/assets/img/possvm-logo.png" /></p>

<p>You can get the code and test data from these repositories:</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Link</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/possvm-orthology"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Source code and manual</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/possvm-orthology-benchmarking"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Test data from <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab234/6342420" target="_blank">Grau-Bové and Sebé-Pedrós, MBE 2021</a>.</td>
    </tr>
  </tbody>
</table>

<h2 id="github-repositories">Github repositories</h2>

<p>Code and instructions for the following projects:</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Link</th>
      <th style="text-align: left">Publication</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/recoded-mixture-models"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Protein mixture models for common amino-acid recoding schemes (e.g. Dayhoff), for use in IQ-TREE.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/ace1-anopheles-report"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Resistance to pirimiphos-methyl in West African <em>Anopheles</em> is spreading via duplication and introgression of the <em>Ace1</em> locus <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009253" target="_blank">(Grau-Bové <em>et al.</em>, Plos Genetics 2021)</a>.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/rdl-Agam-evolution" target="_blank"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Evolution of the insecticide target <em>Rdl</em> in African <em>Anopheles</em> is driven by interspecific and interkaryotypic introgression <a href="https://academic.oup.com/mbe/article/37/10/2900/5843798" target="_blank">(Grau-Bové <em>et al.</em>, MBE 2020)</a>.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/elifesciences-publications/oenocytes-agam" target="_blank"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Isolation and transcriptomic analysis of <em>Anopheles gambiae</em> oenocytes enables the delineation of hydrocarbon biosynthesis <a href="https://elifesciences.org/articles/58019" target="_blank">(Grigoraki <em>et al.</em>, eLife 2020)</a>.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/paraphelidium2018" target="_blank"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Global transcriptome analysis of the aphelid <em>Paraphelidium tribonemae</em> supports the phagotrophic origin of fungi <a href="https://www.nature.com/articles/s42003-018-0235-z" target="_blank">(Torruella <em>et al.</em>, Communications Biology 2018)</a>.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/alternativesplicing2018" target="_blank"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Origin of exon skipping-rich transcriptomes in animals driven by evolution of gene architecture <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1499-9" target="_blank">(Grau-Bové <em>et al.</em>, Genome Biology 2018)</a>. Genome annotation data and list of species.</td>
    </tr>
  </tbody>
</table>

<h2 id="data">Data</h2>

<p>Data from <strong>Dynamics of genomic innovation in the unicellular ancestry of animals</strong> <a href="https://elifesciences.org/articles/26036" target="_blank">(Grau-Bové <em>et al.</em>, eLife 2017)</a>.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Link</th>
      <th style="text-align: left">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><a href="https://github.com/xgrau/dynamicsinnovation2017" target="_blank"><i class="fa fa-external-link"></i> <i class="fa fa-github"></i></a></td>
      <td style="text-align: left">Supplementary data and figures. Includes phylogenomic alignments.</td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://figshare.com/articles/Genome_-_Corallochytrium_limacisporum/5426470" target="_blank"><i class="fa fa-external-link"></i> Figshare</a></td>
      <td style="text-align: left">Genome assembly and annotation of <em>Corallochytrium limacisporum</em></td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://figshare.com/articles/Genome_-_Chromosphaera_perkinsii/5426494" target="_blank"><i class="fa fa-external-link"></i> Figshare</a></td>
      <td style="text-align: left">Genome assembly and annotation of <em>Chromosphaera perkinsii</em></td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://figshare.com/articles/Genome_-_Ichthyophonus_hoferi/5426488" target="_blank"><i class="fa fa-external-link"></i> Figshare</a></td>
      <td style="text-align: left">Genome assembly and annotation of <em>Ichthyophonus hoferi</em></td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://figshare.com/articles/Genome_-_Abeoforma_whisleri_/5426458" target="_blank"><i class="fa fa-external-link"></i> Figshare</a></td>
      <td style="text-align: left">Genome assembly and annotation of <em>Abeoforma whisleri</em></td>
    </tr>
    <tr>
      <td style="text-align: center"><a href="https://figshare.com/articles/Genome_-_Pirum_gemmata/5426506" target="_blank"><i class="fa fa-external-link"></i> Figshare</a></td>
      <td style="text-align: left">Genome assembly and annotation of <em>Pirum gemmata</em></td>
    </tr>
  </tbody>
</table>


</article>

</div>

  <footer class="footer">
  
    <a href="https://github.com/xgrau/" class="menu-link" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
  
    <a href="https://orcid.org/0000-0003-1978-5824" class="menu-link" target="_blank"><i class="fa fa-info-circle" aria-hidden="true"></i></a>
  
    <a href="https://twitter.com/xgraubove" class="menu-link" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
  
    <a href="feed.xml" class="menu-link" target="_blank"><i class="fa fa-rss-square" aria-hidden="true"></i></a>
  
  <div class="post-date"><a href="/">Xavier Grau-Bové</a> | <a href="https://lenpaul.github.io/Millennial/">Millennial theme by Paul Le</a></div>
</footer>

</div>

</body>
</html>
