---
layout: misc
title: Data & code
---

## How to cite

You can cite the relevant papers (linked below, for each piece of data). Please & thank you.

## Possvm

***Possvm*** is a python utility that identifies pairs and clusters of orthologous genes from gene phylogenies [(Grau-Bové and Sebé-Pedrós, bioRxiv 2021)](https://www.biorxiv.org/content/10.1101/2021.05.03.442399v1){:target="_blank"}. It takes advantage of the [species overlap algorithm](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2007-8-6-r109) implemented in the [*ETE* toolkit](http://etetoolkit.org/docs/latest/tutorial/tutorial_phylogeny.html#species-overlap-so-algorithm) to parse the phylogeny and identify orthologous gene pairs, and [*MCL* clustering](https://micans.org/mcl/) for orthogroup identification. 

<img align="centre" width="300" src="/assets/img/possvm-logo.png">

You can get the code and test data from these repositories:

| Links | Publication |
| :--: | :---------- |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/possvm-orthology) | Source code |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/possvm-orthology-benchmarking) | Test data from [Grau-Bové and Sebé-Pedrós, bioRxiv 2021](https://www.biorxiv.org/content/10.1101/2021.05.03.442399v1){:target="_blank"}. |

## Github repositories

Code and instructions for the following projects:


| Links | Publication |
| :--: | :---------- |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/ace1-anopheles-report) | Resistance to pirimiphos-methyl in West African *Anopheles* is spreading via duplication and introgression of the *Ace1* locus [(Grau-Bové *et al.*, Plos Genetics 2021)](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009253){:target="_blank"}. |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/rdl-Agam-evolution){:target="_blank"} | Evolution of the insecticide target *Rdl* in African *Anopheles* is driven by interspecific and interkaryotypic introgression [(Grau-Bové *et al.*, MBE 2020)](https://academic.oup.com/mbe/article/37/10/2900/5843798){:target="_blank"}. |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/elifesciences-publications/oenocytes-agam){:target="_blank"} | Isolation and transcriptomic analysis of *Anopheles gambiae* oenocytes enables the delineation of hydrocarbon biosynthesis [(Grigoraki *et al.*, eLife 2020)](https://elifesciences.org/articles/58019){:target="_blank"}. |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/paraphelidium2018){:target="_blank"} | Global transcriptome analysis of the aphelid *Paraphelidium tribonemae* supports the phagotrophic origin of fungi [(Torruella *et al.*, Communications Biology 2018)](https://www.nature.com/articles/s42003-018-0235-z){:target="_blank"}. |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/alternativesplicing2018){:target="_blank"} | Origin of exon skipping-rich transcriptomes in animals driven by evolution of gene architecture [(Grau-Bové *et al.*, Genome Biology 2018)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1499-9){:target="_blank"}. Genome annotation data and list of species. |

## Data

Data from **Dynamics of genomic innovation in the unicellular ancestry of animals** [(Grau-Bové *et al.*, eLife 2017)](https://elifesciences.org/articles/26036){:target="_blank"}.

| Links | Description |
| :--: | :---------- |
| [<i class="fa fa-external-link"></i> <i class="fa fa-github"></i>](https://github.com/xgrau/dynamicsinnovation2017){:target="_blank"} | Supplementary data and figures. Includes phylogenomic alignments. |
| [<i class="fa fa-external-link"></i> Figshare](https://figshare.com/articles/Genome_-_Corallochytrium_limacisporum/5426470){:target="_blank"} | Genome assembly and annotation of *Corallochytrium limacisporum* |
| [<i class="fa fa-external-link"></i> Figshare](https://figshare.com/articles/Genome_-_Chromosphaera_perkinsii/5426494){:target="_blank"} | Genome assembly and annotation of *Chromosphaera perkinsii* |
| [<i class="fa fa-external-link"></i> Figshare](https://figshare.com/articles/Genome_-_Ichthyophonus_hoferi/5426488){:target="_blank"} | Genome assembly and annotation of *Ichthyophonus hoferi* |
| [<i class="fa fa-external-link"></i> Figshare](https://figshare.com/articles/Genome_-_Abeoforma_whisleri_/5426458){:target="_blank"} | Genome assembly and annotation of *Abeoforma whisleri* |
| [<i class="fa fa-external-link"></i> Figshare](https://figshare.com/articles/Genome_-_Pirum_gemmata/5426506){:target="_blank"} | Genome assembly and annotation of *Pirum gemmata* |


